tgui/packages/tgui/interfaces/Uplink.js

```diff
@@ -96,49 +124,65 @@ const ItemsPage = (_properties, context) => {
           <Button
             content="Random Item"
             icon="question"
-            onClick={() => act('buyRandom')}
+            onClick={() => act("buyRandom")}
           />
           <Button
             content="Refund Currently Held Item"
             icon="undo"
-            onClick={() => act('refund')}
+            onClick={() => act("refund")}
           />
         </Fragment>
       }>
+      <Input
+        fluid
+        mb={1}
+        placeholder="Search equipment"
```

Comment:
Вроде бы `Find` уместнее, нет?
```suggestion
        placeholder="Find equipment"
```

Reply:
Ну не знаю, во второй вкладке серч использовался